commands-yml/commands/device/app/app-state.yml

```diff
@@ -39,8 +40,8 @@ example_usage:
 client_docs:
   java: "https://appium.github.io/java-client/io/appium/java_client/InteractsWithApps.html#queryAppState-java.lang.String-"
   python: "https://github.com/appium/python-client/blob/master/appium/webdriver/extensions/applications.py"
-  javascript_wdio: "https://webdriver.io/docs/api/appium.html" # TODO wdio documentation link
-  javascript_wd: "https://github.com/admc/wd/blob/master/lib/commands.js" # TODO wd documentation link
+  javascript_wdio: "https://webdriver.io/docs/api/appium.html#queryappstate"
+  javascript_wd: "https://webdriver.io/docs/api/appium.html#queryappstate" # TODO wd documentation link
```

Comment:
I don't think this `wd` docs link should go to `webdriverio`.

Reply:
let me fix that